evals/agentic_replay_engines.py

```
@@ -0,0 +1,372 @@
+def version_command(arm: EngineArm) -> list[str]:
+    if arm.engine == "sglang":
+        return [
+            str(arm.executable),
+            "-c",
+            "import importlib.metadata; print(importlib.metadata.version('sglang'))",
+        ]
+    return [str(arm.executable), "--version"]
+
+
+def engine_version(arm: EngineArm) -> str:
+    result = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)